dv/uvm/core_ibex/scripts/check_logs.py

```diff
@@ -54,6 +62,7 @@ def compare_test_run(trr: TestRunResult) -> TestRunResult:
         process_ibex_sim_log(trr.rtl_trace, trr.dir_test/'rtl_trace.csv')
     except (OSError, RuntimeError) as e:
         trr.passed = False
+        trr.failure_mode = Failure_Modes.LOG_ERROR
```

Comment:
I think this should be a different failure mode to the LOG_ERROR above. The one above is a successfully read log that has reported a failure. This is a failure to properly read that log. Maybe this is PARSE_ERROR too or a new category?

Reply:
Pushed a new commit with the modes updated to FILE_ERROR and LOG_ERROR, with comments on each. Does that make more sense?